src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__frame-photo">
+        <img class="card__photo" src="images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock">
+      </div>
+
+      <h2 class="card__name-model">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <h3 class="card__code">
+        Product code: 195434
+      </h3>
+
+      <p class="card__review">
+        <span class="card__review-text">
+          Reviews: 5
+        </span>
+        <span class="stars stars--how-many--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </span>
+      </p>
+
+      <p class="card__price-content">
+        <span class="card__price-text">
+          Price:
+        </span>
+        <strong class="card__price-usd">
+          $2,199
+        </strong>
+      </p>
+      
+      <lable class="card__button-link-buy">
```

Comment:
Replace 'lable' with the correct tag 'label' but this wrapper is redundant. You can just apply styles to your link `a`